### PR TITLE
PavloShevchuk task 2

### PR DESCRIPTION
**Pull request by PashaShevchuk**
I completed task-3, task-4 and task-5

### CODE REVIEW

File: src/components/comment/Comment.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import './Comment.scss';
+
+export function PostsComments({comment}) {
+    if (comment.length === 0) return null;
+    return (
```
Comment (AlyonaPianykh) [inline src/components/comment/Comment.js:7]: Very nice!

File: src/components/comment/Comment.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        comment.map(item => {
+            return (
+                <div className='post-comment'>
```
Comment (AlyonaPianykh) [inline src/components/comment/Comment.js:10]: I suppose here also should be `key` added

**Comment by AlyonaPianykh** [comment]
Review done
Good!

File: src/components/comment/Comment.js (hunk 2 above)
Comment (PashaShevchuk) [reply src/components/comment/Comment.js:10]: I added key.

**Pull request closed without merging**